### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by TundraApp** [anidl/multi-downloader-nx#64] [Q&A] [answered]
Can someone explain how to use this? If you could I would be willing to PayPal you $10 if your way works

**Comment by izu-co** [comment, marked as answer]
You don't have to pay for help. 

Note: The prerequirements must be met. 

First, to get the version with fixed downloads head over to [my fork](https://github.com/AnAppleforlife/funimation-downloader-nx) (for the new episodes) an click on 'code'→'download zip'.

Once downloaded unzip the folder anywhere you like. Open a terminal (cmd on Windows) and move into the downloaded directory. Type 'npm install' and wait for the command to finish. You may now log in by using 'node funi.js --auth'.

Afterwards you can search for a show by using 'node funi.js --search "[Insert you search]"'. The result should look like this: [#123456] Super Duper show.

Take the id (123456) and insert it into the next command like so 'node funi.js -s 123456'. Add an '-e 1,2,3,4-6' to select the episodes (note that you should insert the episodes you want). You can select the quality by adding -q '[1-10]'.

If you run into any trouble, just open an issue.

**Reply by TundraApp** [reply]
Question what is the ID and how do I get it?

**Reply by izu-co** [reply]
> Afterwards you can search for a show by using 'node funi.js --search "[Insert you search]"'. The result should look like this: [#123456] Super Duper show.
> 
> Take the id (123456) and insert it into the next command like so 'node funi.js -s 123456'.

**Reply by TundraApp** [reply]
> Afterwards you can search for a show by using 'node funi.js --search "[Insert you search]"'. The result should look like this: [#123456] Super Duper show.
> 
> Take the id (123456) and insert it into the next command like so 'node funi.js -s 123456'.

so is the ID 123456?

**Reply by izu-co** [reply]
Yes. Of course, you have to change the ID to the one you are getting.

**Reply by TundraApp** [reply]
where do I get the ID and thank you for helping me

**Reply by TundraApp** [reply]
> Yes. Of course, you have to change the ID to the one you are getting.

what I'm trying to ask is where the ID is shown or how I get the ID

**Reply by flashpc** [reply]
easiest way to get the id is to use the search function, say you wanted to look for my hero academia you would use the command
node funi --search "my hero"
it will search the entire site for videos matching that term, 
![myhero](https://user-images.githubusercontent.com/26695376/117372736-ed69ff00-aec1-11eb-9b7f-b8a1f4f76d29.jpg)
as you can see the id for my hero academia is 124389

now when you want to download an episode you would use the command
node funi -s 124389 -e 67
this would download episode 67 from my hero academia

**Reply by TundraApp** [reply]
> easiest way to get the id is to use the search function, say you wanted to look for my hero academia you would use the command
> node funi --search "my hero"
> it will search the entire site for videos matching that term,
> ![myhero](https://user-images.githubusercontent.com/26695376/117372736-ed69ff00-aec1-11eb-9b7f-b8a1f4f76d29.jpg)
> as you can see the id for my hero academia is 124389
> 
> now when you want to download an episode you would use the command
> node funi -s 124389 -e 67
> this would download episode 67 from my hero academia

ok thank you so much

**Reply by TundraApp** [reply]
> You don't have to pay for help.
> 
> Note: The prerequirements must be met.
> 
> First, to get the version with fixed downloads head over to [my fork](https://github.com/AnAppleforlife/funimation-downloader-nx) (for the new episodes) an click on 'code'→'download zip'.
> 
> Once downloaded unzip the folder anywhere you like. Open a terminal (cmd on Windows) and move into the downloaded directory. Type 'npm install' and wait for the command to finish. You may now log in by using 'node funi.js --auth'.
> 
> Afterwards you can search for a show by using 'node funi.js --search "[Insert you search]"'. The result should look like this: [#123456] Super Duper show.
> 
> Take the id (123456) and insert it into the next command like so 'node funi.js -s 123456'. Add an '-e 1,2,3,4-6' to select the episodes (note that you should insert the episodes you want). You can select the quality by adding -q '[1-10]'.
> 
> If you run into any trouble, just open an issue.

hey sorry for bothering you again but the files are downloading as a .ts file is there a way to make it a .mp4 file?

**Reply by izu-co** [reply]
If you have ffmpeg installed and set the path in /config/bin-path.yml it will do it automatically

**Reply by TundraApp** [reply]
ok thank you so much

**Comment by edward8e** [comment]
Hi I did my best to follow the instructions in the readme, however, after I run:
`npm i`
`npm run tsc`

I change into the lib directory and see all the JavaScript files that were generated, however, I don't know what I'm missing to run them. I am able to use the version listed by @izu-co [izu-co](https://github.com/izu-co) above, however, when I build it myself I'm not able to run it. Am I missing something?

**Comment by KuroProjects** [comment]
hi 
i want to use thee same as for crunchyroll please help me with that

**Reply by izu-co** [reply]
Hey there. This thread is already answered, so open up a new one instead :)
